agent/pkg/crane/README.md

```diff
@@ -95,29 +69,6 @@ Run installed tool with like so
 golangci-lint run
 ```
 
-### Changelog
-
-Install the generator
-```
-go get -u github.com/git-chglog/git-chglog/cmd/git-chglog
-```
-
-Usage
-```
-git-chglog -o CHANGELOG.md
-```
-
-```
-git tag -a v1.x.x
-git push --tags
-```
-
-In order to draft a new release: 
-- create a new release tag on develop
-- generate changelogs
-
-See [CHANGELOG.md](CHANGELOG.md) 
-
```

Comment:
Where is this available now?

Reply:
In the root folder.

Reply:
Sorry, that was not phrased clearly. Where is the answer to this: `how to generate changelogs?`

Reply:
I removed it, I think we have to set up an action or something like that for it. If you want I can add it back to the root folder README.md if needed.

Reply:
Until we don't have it automatized, it would worth keeping the hints somewhere.

Reply:
Added back in the root folder README.md